jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/db2/DB2SqlDialect.java

```diff
@@ -107,4 +107,10 @@ public NullSorting getDefaultNullSorting() {
     public String getStringLiteral(final String value) {
         return "'" + value.replace("'", "''") + "'";
     }
+
+    @Override
+    public void validateProperties() throws PropertyValidationException {
```

Comment:
Superfluous since name check is dead code.

Reply:
Decided to leave it be